graylog2-web-interface/src/pages/ShowEventNotificationPage.tsx

```diff
@@ -77,8 +77,8 @@ const ShowEventDefinitionPage = () => {
           path: DocsHelper.PAGES.ALERTS,
         }}>
         <span>
-          Notifications alert you of any configured Event when they occur. Graylog can send Notifications directly to
-          you or to other systems you use for that purpose.
+          Notifications alert you of any configured Event when they occur. Notifications can be send directly to you or
```

Comment:
```suggestion
          Notifications alert you of any configured Event when they occur. Notifications can be sent directly to you or
```
Should be consolidated to a shared component.